controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -668,6 +668,7 @@ stream {
         set $proxy_upstream_name "{{ buildUpstreamName $server.Hostname $all.Backends $location }}";
 
         {{ $ing := (getIngressInformation $location.Ingress $path) }}
+        {{/* $ing.Metadata contains the Ingress metadata */}}
```

Comment:
`$ing.Annotations contains the Ingress annotations`?

Reply:
yes